delfi_attributes.json

```diff
@@ -0,0 +1,862 @@
+		"CONDITION": {
+			"osm_tags": {
+				"railway": "crossing"
+			},
+			"osm_element": "Node"
+		},
+		"TAGS": {}
+	},
+	{
+		"DELFI_ID": 2050,
+		"LABEL": "Unbefestigter Bodenbelag",
+		"CONDITION": {
+			"osm_tags": {
+				"highway": ["footway", "path"]
+			},
+			"osm_element": "OpenWay"
+		},
+		"TAGS": {
+			"surface": ["paved", "unpaved", "asphalt", "concrete", "paving_stones", "sett", "cobblestone", "metal", "wood", "compacted", "fine_gravel", "pebblestone", "earth", "grass", "ground", "gravel", "mud", "sand", "snow", "ice", "salt", "woodchips", "grass_paver", "gravel_turf", "unhewn_cobblestone", "shell", "grasscrete", "rock", "clay", "debris", "dirt", "grass_paving", "plastic", "tar_and_chip", "woodchips"]
+		}
+	},
+	{
+		"DELFI_ID": 2070,
+		"LABEL": "Bodenindikatoren vorhanden",
+		"CONDITION": [
+			{
+				"osm_tags": {
+					"highway": ["footway", "path", "steps"]
+				}
+			},
+			{
+				"osm_tags": {
+					"railway": "platform"
+				}
+			},
+			{
+				"osm_tags": {
+					"public_transport": "platform"
+				}
+			}
+		],
+		"TAGS": {
+			"tactile_paving": ["yes", "no", "contrasted"]
+		}
+	},
+	{
+		"DELFI_ID": 2090,
+		"LABEL": "Aufzug vorhanden",
+		"CONDITION": {
+			"osm_tags": {
+				"highway": "elevator"
+			}
+		},
+		"TAGS": {}
+	},
+	{
+		"DELFI_ID": 2090,
+		"LABEL": "Türbreite Aufzug",
+		"CONDITION": {
+			"osm_tags": {
+				"highway": "elevator"
+			}
+		},
+		"TAGS": {
+			"entrance": [],
+			"width": []
+		}
+	},
+	{
+		"DELFI_ID": 2093,
+		"LABEL": "Grundflächenlänge",
+		"CONDITION": {
+			"osm_tags": {
+				"highway": "elevator"
+			}
+		},
+		"TAGS": {
+			"length": []
+		}
+	},
+	{
+		"DELFI_ID": 2094,
+		"LABEL": "Grundflächenbreite",
+		"CONDITION": {
+			"osm_tags": {
+				"highway": "elevator"
+			}
+		},
+		"TAGS": {
+			"width": []
+		}
+	},
+	{
+		"DELFI_ID": 2092,
+		"LABEL": "Grundfläche des Aufzugs",
+		"CONDITION": {
+			"osm_tags": {
+				"highway": "elevator"
+			}
+		},
+		"TAGS": {
+			"width": [],
+			"length": []
+		}
+	},
+	{
+		"DELFI_ID": 2095,
+		"LABEL": "Verbindungsfunktion",
+		"CONDITION": {
+			"osm_tags": {
+				"highway": "elevator"
+			}
+		},
+		"TAGS": {
+			"level": []
+		}
+	},
+	{
+		"DELFI_ID": 2100,
+		"LABEL": "Stufe vorhanden",
+		"CONDITION": [
+			{
+				"osm_tags": {
+					"highway": "steps",
+					"step_count": "1"
+				}
+			},
+			{
+				"osm_tags": {
+					"barrier": "kerb",
+					"kerb": true
+				}
+			}
+		],
+		"TAGS": {}
+	},
+	{
+		"DELFI_ID": 2101,
+		"LABEL": "Stufenhöhe",
+		"CONDITION": [
+			{
+				"osm_tags": {
+					"highway": "steps",
+					"step_count": "1"
+				}
+			},
+			{
+				"osm_tags": {
+					"barrier": "kerb",
+					"kerb": true
+				}
+			}
+		],
+		"TAGS": [
+			{
+				"step_height": []
+			},
+			{
+				"kerb:height": []
+			}
+		]
+	},
+	{
+		"DELFI_ID": 2110,
+		"LABEL": "Treppe vorhanden",
+		"CONDITION": {
+			"osm_tags": {
+				"highway": "steps"
+			},
+			"osm_element": "OpenWay"
+		},
+		"TAGS": {
+			"incline": []
+		}
```

Comment:
Hm... yeah, I guess. My thought proccess was that since there is no condition for "Richtung der Treppe verzeichnet", I would make a Treppe only count as sufficiently mapped if the incline is recorded. Maybe the incline-Condition could be added to the "Verbindungsfunktion der Treppe"-check? So that a Treppe only counts as having its Verbindungsfunktion mapped if it has levels _and_ an incline?